src/anomalib/models/efficientad/torch_model.py

```diff
@@ -0,0 +1,314 @@
+"""Torch model for student, teacher and autoencoder model in EfficientAD"""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+import random
+from pathlib import Path
+from typing import Union
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor, nn
+from torchvision import transforms
+
+logger = logging.getLogger(__name__)
+
+
+def imagenet_norm_batch(x):
+    mean = torch.tensor([0.485, 0.456, 0.406])[None, :, None, None].cuda()
+    std = torch.tensor([0.229, 0.224, 0.225])[None, :, None, None].cuda()
```

Comment:
I'd be in favour of using `to(x.device)` as some users might want to run it on the CPU